src/main/java/net/dv8tion/jda/bot/sharding/DefaultShardManagerBuilder.java

```diff
@@ -82,6 +82,19 @@
      * before calling {@link net.dv8tion.jda.bot.sharding.DefaultShardManagerBuilder#build() build()}.
      */
     public DefaultShardManagerBuilder() {}
+    
+    /**
+     * Creates a DefaultShardManagerBuilder with the given token.
+     * <br>This is equivalent to using the constuctor
+     * {@link net.dv8tion.jda.bot.sharding#DefaultShardManagerBuilder() DefaultShardManager()}
```

Comment:
Is this link actually working? Shouldn't this be `...sharding.DefaultShardManagerBuilder#DefaultShar...()`?

Reply:
Also the alias is wrong

Reply:
Fixed the alias. In theory, the constructor link works, haven't had a chance to try it out (I've been working from the web editor)